modules/specialized-guides/pages/salt/salt-formula-liberate.adoc

```diff
@@ -0,0 +1,309 @@
+// FIXME: where appropriate, use the "Confirm with" expression.
+
+[[liberate-formula]]
+= Liberate Formula
+
+The liberate formula migrates systems from EL clients such as {almalinux}, {centos} 7, {oracle} 9, {rocky} 9, or even {rhel} 9 to {sll}.
+With this formula the conversion will take place during the client onboarding on {productname}.
+
+
+
+== Install the formula
+
+Install the formula with:
+
+----
+zypper in liberate-formula`
+----
+
+
+
+== Configure {productname}
+
+
+
+To provide all the software channels for {sll} on the {productname} Server, proceed as follows.
+
+
+
+.Procedure: Providing SUSE Customer Center credentials
+
+. Sign in to {scc} at https://scc.suse.com.
+
+. Navigate to [guimenu]``My Organization``, and select your organization.
+
+. Navigate to menu:Users[Organization Credentials] and take note of your organization username and password.
+
+. In the {productname} {webui}, navigate to menu:Admin[Setup Wizard > Organization Credentials] to add the credentials to your {productname} Server.
+
+. Click btn:[Add new credential], and enter the {scc} username and password noted in a previous step.
+
+
+.Procedure: Synchronizing the SLL/SLES-ES channels:
+
+. In the {productname} {webui}, navigate to menu:Admin[Setup Wizard > Products]
+. Select the {sll} Channels that you will use:
++
+
+* EL7: `SUSE Linux Enterprise Server with Expanded Support 7 x86_64`
+* EL8: `RHEL or SLES ES or CentOS 8 Base`
+* EL9: `RHEL and Liberty 9 Base`
+
+. Click the top right button btn:[Add products].
+
+
+Initial sychronization can take considerable time. You can check progress by accessing the server machine via SSH and monitoring the logs using:
+
+----
+tail -f /var/log/rhn/reposync/*
+----
+
+
+.Procedure: Creating one Activation Key per {sll} parent channel
+
+. Note: Activation Keys are the way to register systems and automatically assign them to the required software and configuration channels corresponding to them.
+. In the {productname} {webui}, navigate to menu:Systems[Activation Keys], and click the top right button btn:[Create key].
+. In the [guimenu]``Activation Key`` dialog, set the fields:
++
+Description::
+With some text describing the activation key
+Key::
+With the identifier for the key.  For example [literal``sll9-default`` for EL 9 systems.
+Note: Keys will have a numeric prefix depending on the organization, so that they are unique.
+Usage::
+Leave blank
+Base Channel::
+Select one base channel:
++
+
+* EL7: `RHEL x86_64 Server 7`
+* EL8: `RHEL8-Pool for x86_64`
+* EL9: `EL9-Pool for x86_64`
+
+Child Channel::
+Use [literal]``include recommended`` where available or select all if unavailable
```

Comment:
It turned out that the suggestion to include recommended is not quite correct as discussed in this Slack thread https://suse.slack.com/archives/C02D78LLS04/p1707753031169719.

I think the safest approach is to always include all channels, unless the customer knows exactly what channels are needed.

@rjmateus What do you think?

Reply:
I agree with Thomas, we should say to add all channels. Another option could be say that we need client tools channels and update channels, but this option can get outdated faster.

Reply:
thanks for feedback, I changed it accordingly.